packages/web/src/app/icons.css

```
@@ -0,0 +1,325 @@
+  align-items: center;
+  justify-content: center;
+  width: 1.5em;
+  height: 1.5em;
+  min-width: 24px;
+  min-height: 24px;
+  vertical-align: -0.25em;
+  border-radius: 50%;
+  border: 1.5px solid #000;
+  background-color: #fff;
+  color: #000;
+  font-family: "arpona", serif; /* Typekit — same as .tk-arpona */
```

Stylelint (17.14.0)

[error] Expected no quotes around "arpona" (font-family-name-quotes)

(font-family-name-quotes)